File: C/7.c
#include <stdio.h>

int main(){
    // int num = 11;
    // (num % 2 == 0) ? printf("Even Number") : printf("Odd Number");

    // int a = 100, b = 200, c = 30;
    // int res = (a > b) ? (a > c ? a : b) : (b > c ? b : c);
    // printf("Bigger value is: %d", res);


    int a, b;
    char op;

    printf("Enter first number: ");
    scanf("%d", &a);

    printf("Enter second number: ");
    scanf("%d", &b);

    printf("Enter operator (+,/,-,*): ");
    scanf(" %c", &op);

    switch(op){
        case '+':
            printf("a + b is: %d", a + b);
            break;
        case '-':
            printf("\na - b is: %d", a - b);
            break;
        case '*':
            printf("\na * b is: %d", a * b);
            break;
        case '/':
            printf("\na / b is: %d", a / b);
            break;
        default:
            printf("\nInvalid operator");
            break;
    }
}

File: python/7.py
for i in range(1,6):
    for j in range(1,6):
        print(j, end=" ")
    print()
    
print("-----------")

i = 1
while i <= 5:
    i += 1
    j = 1
    while j <= 5:
        print(j, end=" ")
        j += 1
    print()
    
print("-----------")
    
for i in range(1,6):
    for j in range(i,6):
        print(j, end=" ")
    print()
    
print("-----------")

for i in range(1,6):
    for j in range(1,i+1):
        print(j, end=" ")
    print()
    
print("-----------")

num = 1

for i in range(1,6): 
    for j in range(1,i+1):
        print(num, end=" ")
        num += 1
    print()
    
print("-----------")

for i in range(1,6):
    for k in range(5,i,-1):
        print(" ", end=" ")
    for j in range(1, i*2):
        print(j, end=" ")
    print()